WalletWasabi.Backend/WalletWasabi.Backend.csproj

```diff
@@ -2,6 +2,8 @@
 
 	<PropertyGroup>
 		<DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
+		<AnalysisLevel>latest</AnalysisLevel>
+		<LangVersion>11</LangVersion>
```

Comment:
These changes are wrong. We moved to NET8.0 and set C# version to 12. The same change is in all the csproj files.